app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[0] == "cp":
```

Comment:
`... and command_list[-1] != command_list[-2]:`

Reply:
fixed